src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-config-details/rgw-config-details.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Input, OnChanges } from '@angular/core';
+
+const keyMapping = {
+  auth: 'Authentication Method',
+  encryption_type: 'Encryption Type',
+  backend: 'Backend',
+  prefix: 'Prefix',
+  namespace: 'Namespace',
+  secret_engine: 'Secret Engine',
+  addr: 'Address',
+  token_file: 'Token File',
+  ssl_cacert: 'SSL CA Certificate',
+  ssl_clientcert: 'SSL Client Certificate',
+  ssl_clientkey: 'SSL Client Key',
+  verify_ssl: 'Verify SSL'
+};
```

Comment:
why don't we use an interface that can store these details and we just have to use that interface wherever we want?